components/layout.js

```diff
@@ -24,12 +24,16 @@ const layoutStyle = {
   border: '1px solid #DDD'
 };
 
-const Layout = ({ children }) => (
+const Layout = ({ children, clientHintDeviceMemory }) => {
+  return (
   <div style={layoutStyle}>
     <Head>
       <title>Adaptive Animation</title>
     </Head>
-    <MemoryStatus />
+    { clientHintDeviceMemory ? 
+      <div style={{'display':'flex'}}><h3>SSR:</h3><h4>[CH DeviceMemory: {clientHintDeviceMemory}GByte]</h4></div> : 
```

Comment:
We don't use inline styling -> We can make it as a component too. Every entity with sense can be a component.

Reply:
Also
{ value ? (
    <TrueComponent />
  ) : (
    <FalseComponent />
) }